File: src/main/java/vocadb/notification/reader/client/query/ArtistParticipationStatus.java
package vocadb.notification.reader.client.query;

import com.fasterxml.jackson.annotation.JsonProperty;

public enum ArtistParticipationStatus {
    @JsonProperty("Everything")
    EVERYTHING,
    @JsonProperty("OnlyMainAlbums")
    ONLY_MAIN_ALBUMS,
    @JsonProperty("OnlyCollaborations")
    ONLY_COLLABORATORS;
}

File: src/main/java/vocadb/notification/reader/client/api/enums/ArtistParticipationStatus.kt
package vocadb.notification.reader.client.api.enums

import com.fasterxml.jackson.annotation.JsonValue

enum class ArtistParticipationStatus(@JsonValue val value: String) {
    EVERYTHING("Everything"),
    ONLY_MAIN_ALBUMS("OnlyMainAlbums"),
    ONLY_COLLABORATORS("OnlyCollaborations")
}
